tests/boardfarm_plugins/boardfarm_prplmesh/tests/initial_ap_config.py

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# SPDX-FileCopyrightText: 2020 the prplMesh contributors (see AUTHORS.md)
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+
+from boardfarm.tests import bft_base_test
+
+
+class InitialApConfig(bft_base_test.BftBaseTest):
+    """PrplMesh sample test case, no actual testing, it just passes."""
+
+    def runTest(self):
+        """Main test logic must be implemented here."""
+
+    @classmethod
+    def teardown_class(cls):
+        """Teardown method, optional for boardfarm tests."""
```

Comment:
Same here - the whole commit looks like it was "cut" in the middle. Something wrong with rebase?

Reply:
> Same here - the whole commit looks like it was "cut" in the middle. Something wrong with rebase?

Actually - no.
I've splitted all commits to atomic units as were requested by @arnout.
In that articular case - the dummy testsuite was added, its just passes.